cmd/logcli/main.go

```diff
@@ -229,7 +229,8 @@ func newQuery(instant bool, cmd *kingpin.CmdClause) *query.Query {
 		cmd.Flag("since", "Lookback window.").Default("1h").DurationVar(&since)
 		cmd.Flag("from", "Start looking for logs at this absolute time (inclusive)").StringVar(&from)
 		cmd.Flag("to", "Stop looking for logs at this absolute time (exclusive)").StringVar(&to)
-		cmd.Flag("step", "Query resolution step width").DurationVar(&q.Step)
+		cmd.Flag("step", "Query resolution step width, for metric queries. Evaluate the query at the specified step over the time range.").DurationVar(&q.Step)
+		cmd.Flag("interval", "Query interval, for log queries. Return entries at the specified interval, ignoring those between.").DurationVar(&q.Interval)
```

Comment:
I don't want to bikeshed here but I'm worried the difference between step and interval isn't totally clear and might be hard to remember. Would `log_step` or `entry_step` be easier here?

Reply:
That would be more confusing to me personally.  

I would prefer to work on making the description clearer if we can do so.

Reply:
I could be wrong, but I'm concerned about having naming conventions that requires you to pull up the docs to differentiate between the two. I'm okay with this as-is, but I think it's something to think about.